README.md

```diff
@@ -0,0 +1,122 @@
+# Welcome to the Bus Stop Census GitHub Page! 
+
+## Disclaimer
+Please read this from start to finish before exploring the repository. This README describes the tools needed to run the code included in this repository, the files contained in the repository, and any miscellaneous details that will help you understand what is in this repository. Ignore at your own risk. 
+
+## What is the purpose of this repository?
+The purpose of this repository is to house the data, code, figures, and products for the Bus Stop Census data anaylsis. MARTA Army has been collecting data on MARTA bus stops from February 4th, 2020 to Dec 31st, 2020. With data collection over, the task remains to clean, validate, and analyze the data. The repository documents this process and provides an organized and replicable framework for the public to use for their own analyses.
+
+## R and RStudio Set Up Instructions
+To get started, follow these instructions to install R and RStudio adapted from [Copyright (c) Software Carpentry](http://software-carpentry.org).
+
+### R and RStudio
+- Download and install [R, a free software environment for statistical computing and graphics](https://www.r-project.org/) from [CRAN](https://cran.rstudio.com/), the Comprehensive R Archive Network. It is highly recommended to install a precompiled binary distribution for your operating system – use the links up at the top of the CRAN page linked to above!
+
+- Install RStudio's IDE (stands for integrated development environment), a powerful user interface for R: http://www.rstudio.com/ide/download/
+  - RStudio includes a text editor, so you do not have to install another stand-alone editor.
+  - RStudio includes an interface to Git and GitHub. You will still need to install Git (covered below) but RStudio provides a basic GUI for interacting with Git(Hub).
+  
+- Install [Git](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git)
+  - To access this repository, you will need to create an account on [GitHub.com](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/signing-up-for-a-new-github-account).
```

Comment:
Not needed. IMO having git is implied, and a GitHub account is not required to simply clone the repo.

Reply:
Agreed. Removing